service/src/main/resources/application.yml

```diff
@@ -86,15 +86,11 @@ otel:
   sdk:
     disabled: false # set to true to disable all open telemetry features
 
-  springboot:
-    resource:
-      attributes:
-        service:
-          name: ${spring.application.name}
-
-  instrumentation:
-    logback-appender:
-      enabled: false
```

Comment:
I believe this is no longer necessary, with the latest tcl changes in place.

Reply:
Was this also copied from another PR?

Reply:
Not copied, but this was the original fix which disabled the appender, before we had a fix in tcl.